src/explore-education-statistics-admin/src/components/ReleaseServiceStatus.tsx

```diff
@@ -12,15 +12,13 @@ interface Props {
 }
 
 type Stage =
-  | 'Scheduled'
-  | 'NotStarted'
-  | 'Invalid'
-  | 'Failed'
-  | 'Cancelled'
   | 'Validating'
+  | 'Cancelled'
+  | 'Complete'
+  | 'Failed'
   | 'Queued'
-  | 'Started'
-  | 'Complete';
+  | 'NotStarted'
```

Comment:
Think you've misunderstood what Ben's asking you to do.

This current type is flawed because it implies that every stage will potentially have any one of those strings, when in reality we have stages like `overallStage` that doesn't have `NotStarted`.

He's given you the exact list of strings that are allowed for each individual stage. This means that each stage should have a more specific type defined for it. It looks like there is at least some commonality between most of the stages however, so perhaps we just need an `OverallStage` and a `TaskStage` (for content/data/files/publishing).

Reply:
Sorry, didn't spot this earlier. Thanks for explaining this @ntsim